subcommands/targets/offline-update.go

```diff
@@ -78,6 +79,25 @@ func init() {
 		"Allow multiple targets to be stored in the same <dst> directory")
 	offlineUpdateCmd.MarkFlagsMutuallyExclusive("tag", "wave")
 	offlineUpdateCmd.MarkFlagsMutuallyExclusive("prod", "wave")
+	initSignCmd(offlineUpdateCmd)
+}
+
+func initSignCmd(parentCmd *cobra.Command) {
+	signCmd := &cobra.Command{
+		Use:   "sign <path to an offline bundle>",
+		Short: "Sign an offline bundle with a targets role offline key",
+		Long: `Sign an offline bundle with a targets role offline key.
+
+Run this command if your offline update bundle contains production/wave targets.
+In this case, the bundle has to signed by one or more targets role offline keys.
+A number if signatures depends on the threshold number set in the current root role metadata,
+and is printed by this command.`,
+		Run:  doSignBundle,
+		Args: cobra.ExactArgs(1),
+	}
+	signCmd.Flags().StringP("keys", "k", "", "Path to <offline-creds.tgz> used to sign production/wave targets.")
```

Comment:
> Path to <tuf-targets-keys.tgz> used to sign the bundle TUF targets.

I wanted to empathize that the key that has been used for signing waves should be applied here - that's why "used to sign prod & wave". Otherwise, it is not clear what key a user should specify in this command.

Reply:
What about `Path to the <tuf-targets-keys.tgz> key to sign the bundle metadata with; this is the same key used to sign prod & wave TUF targets`.?

Reply:
Yes, it is a good suggestion.
I think two sentences separated by a "dot" make even more sense.